### PR TITLE
Logs Re-Design

### PR DESCRIPTION
**Pull request by dvaldivia**
Screen Shot 2022-03-02 at 9 34 37 PM

Signed-off-by: Daniel Valdivia <18384552+dvaldivia@users.noreply.github.com>

### CODE REVIEW

**Review by prakashsvmx — DISMISSED** [review]
LGTM :+1:

File: portal-ui/src/screens/Console/Logs/ErrorLogs/LogLine.tsx [hunk 1]
```diff
@@ -0,0 +1,229 @@
...
+    fullMessage = messageForConsoleMsg(log);
+  } else if (log.error !== null && log.error.message !== "") {
+    fullMessage = messageForError(log);
+  }
+
+  titleLogMessage = titleLogMessage.replace(tColorRegex, "");
+
+  // let dateStr = (
+  //   <reactMoment.default>{log.time.toString()}</reactMoment.default>
+  // );
```
Comment (bexsoft) [inline portal-ui/src/screens/Console/Logs/ErrorLogs/LogLine.tsx:136]: Please remove these lines

**Review by bexsoft — CHANGES_REQUESTED** [review]
Filter field is not filtering the log lines
Screen Shot 2022-03-03 at 11 34 41

File: portal-ui/src/icons/BoxArrowDown.tsx [hunk 2]
```diff
@@ -0,0 +1,53 @@
+// This file is part of MinIO Console Server
+// Copyright (c) 2021 MinIO, Inc.
```
Comment (jinapurapu) [inline portal-ui/src/icons/BoxArrowDown.tsx:2]:
```suggestion
// Copyright (c) 2022 MinIO, Inc.
```

File: portal-ui/src/icons/BoxArrowUp.tsx [hunk 3]
```diff
@@ -0,0 +1,57 @@
+// This file is part of MinIO Console Server
+// Copyright (c) 2021 MinIO, Inc.
```
Comment (jinapurapu) [inline portal-ui/src/icons/BoxArrowUp.tsx:2]:
```suggestion
// Copyright (c) 2022 MinIO, Inc.
```

File: portal-ui/src/icons/WarnFilledIcon.tsx [hunk 4]
```diff
@@ -0,0 +1,59 @@
+// This file is part of MinIO Console Server
+// Copyright (c) 2021 MinIO, Inc.
```
Comment (jinapurapu) [inline portal-ui/src/icons/WarnFilledIcon.tsx:2]:
```suggestion
// Copyright (c) 2022 MinIO, Inc.
```

**Review by jinapurapu — CHANGES_REQUESTED** [review]

File: portal-ui/src/screens/Console/Logs/ErrorLogs/LogLine.tsx (hunk 1 above)
Comment (dvaldivia) [reply portal-ui/src/screens/Console/Logs/ErrorLogs/LogLine.tsx]: done

**Comment by dvaldivia** [comment]
@jinapurapu done with the 2022 updates :)

**Review by bexsoft — DISMISSED** [review]
LGTM

**Review by jinapurapu — CHANGES_REQUESTED** [review]
Overhang on right
Screenshot from 2022-03-03 12-23-23

some rendering problems in Chrome
Screenshot from 2022-03-03 12-20-43

**Review by jinapurapu — APPROVED** [review]

**Review by Alevsk — APPROVED** [review]

**Pull request merged**